tests/models/conditional_detr/test_modeling_conditional_detr.py

```diff
@@ -159,40 +170,21 @@ def create_and_check_conditional_detr_object_detection_head_model(self, config,
         self.parent.assertEqual(result.logits.shape, (self.batch_size, self.num_queries, self.num_labels))
         self.parent.assertEqual(result.pred_boxes.shape, (self.batch_size, self.num_queries, 4))
 
-    def create_and_check_no_timm_backbone(self, config, pixel_values, pixel_mask, labels):
```

Comment:
Removing this check since now all common tests and tests above use no-timm.

Reply:
The current `main` checks with `timm`. In this PR, no more test will use `timm`, right? 

It looks like we miss some test cases (timm) now, but I am sure you already have this in mind and think it's fine.

Reply:
Integration (slow) tests will still use timm. We can add one using a random tiny timm (not pretrained) if we feel there is a lack of coverage but I think this is enough.